Dockerfile

```diff
@@ -0,0 +1,58 @@
+# This implicitely includes Python 3.10
+FROM node:14-alpine
```

Comment:
I'd prefer to use Ubuntu here if possible. Mainly to keep consistency with the rest of the repositories we have. Also, because I've found different weird compilation issues when working with Alpine and the saving in space is not really huge.

Reply:
I think we're on the same page here and Ubuntu images would be the best way to go.